packages/terra-framework-docs/src/terra-dev-site/doc/date-input/example/DayMonthYearDateInputField.jsx

```diff
@@ -9,16 +9,16 @@ const Example = () => {
   return (
     <React.Fragment>
       <DateInputField
-        legend="Legend text"
+        legend="Event Date"
         name="date-input-value"
         value={value}
         onChange={(event, dateString) => setValue(dateString)}
         displayFormat="day-month-year"
-        error="Error message"
+        error="Please select a valid event date."
```

Comment:
lets update help message in all the examples to have meaningful value like error message and legend text.